deps/chakrashim/core/lib/Runtime/Base/ThreadContext.h

```diff
@@ -309,13 +309,17 @@ class JITTimer
 class ThreadConfiguration
 {
 public:
-    ThreadConfiguration(bool enableExperimentalFeatures)
+    ThreadConfiguration(bool enableExperimentalFeatures, bool enableSimdjsFeature)
     {
         CopyGlobalFlags();
         if (enableExperimentalFeatures)
         {
             EnableExperimentalFeatures();
         }
+        if (!enableSimdjsFeature)
```

Comment:
Since this was temporary, I was not inclined to make it dependent on experimental flag, but on second thought, makes sense to have it that way. I will update the PR.